magento/pwa-studio
# [PWA-1615] Scaffold DEBUG_PROJECT_CREATION does not exclude optional extension packages.

## Description

Steps
1. Clone pwa-studio locally and run yarn install
2. Run DEBUG_PROJECT_CREATION=1 packages/create-pwa/bin/create-pwa
3. Answer all question and go to scaffolded project
4. Run upward server

Expected - Optional extension which are not added should not be included
Actual - If you try navigate around app observe Virtual product sub-type extension, language pack etc are included

## Related Issue





* [[PWA-1615](https://jira.corp.magento.com/browse/PWA-1615)] Scaffold DEBUG_PROJECT_CREATION does not exclude optional extension packages.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Run through repro steps, verify that extensions are no longer in scaffolded app using this flag

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
